x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -331,6 +332,19 @@ export const MLInferenceLogic = kea<
       actions.makeMLModelsRequest(undefined);
       actions.makeMappingRequest({ indexName });
     },
+    setSelectedModelNotPlaceholder: ({ modelId }) => {
```

Comment:
This is kind of weird naming, would `setSelectedModel` be too confusing?

Reply:
`setSelectedModel` would be misleading because we're not setting the entire model object in the configuration, only the `isModelPlaceholderSelected` flag. I agree it's a weird name though, happy to rename it to e.g. `clearModelPlaceholderFlag` or something else if you have any suggestions.

Reply:
Updated the function name to `clearModelPlaceholderFlag`.